app/templates/main/index.html

```diff
@@ -1,15 +1,25 @@
 {% extends 'layouts/base.html' %}
 
 {% block content %}
-    <link rel="stylesheet" type="text/css" href="static/styles/map.css">
+    <link rel="stylesheet" type="text/css" href="static/styles/map.css">   
```

Comment:
nit: why trailing whitespace?
FYI if you're using Sublime, there is a plugin that removes trailing automatically on save.